frontend/src/components/maps/ParcelPopupView.tsx

```diff
@@ -85,12 +85,13 @@ export const ParcelPopupView = (props: IParcelDetailProps | null) => {
               <Col>
                 <Link
                   to={{
-                    pathname: `/mapview/${parcelDetail?.id}`,
+                    pathname: `/mapview`,
```

Comment:
why the path name change?

Reply:
The preference here was to have the parcelId as a search param instead of a query param.